src/content/function-info.js

```diff
@@ -213,3 +213,35 @@ export function getFunctionInfo(fullName) {
          getJSFunctionInfo(fullName) ||
          getFallbackFunctionInfo(fullName);
 }
+
+/**
+ * Strip any function arguments from the given string.
+ *
+ * If the function fails to determine that there are any parentheses to strip
+ * it will return the original string.
+ */
+export function stripFunctionArguments(functionCall) {
+  // Remove known data that can appear at the end of the string
+  const s = functionCall.replace(/ \[clone [^]+\]$/, '').replace(/ const$/, '');
+
+  // Start from the end of the string and assume we are starting on a
+  // right parenthesis.  Then iterate towards the beginning until we
```

Comment:
nit: I think you should test that the last letter is actually the right parenthesis, otherwise the algorithm just fails